cellprofiler/gui/pipelinelistview.py

```diff
@@ -1041,7 +1041,7 @@ def OnData(self, x, y, action):
                 self.data_object.GetReceivedFormat().GetType()
                 == self.pipeline_data_object.GetFormat().GetType()
             ):
-                pipeline_data = self.pipeline_data_object.GetDataHere()
+                pipeline_data = self.pipeline_data_object.GetData().tobytes().replace(b'\00', '')
```

Comment:
I notice you're replacing a byte sequence `b'\00'` with a string - is `pipeline_data` supposed to be a string here or a `bytes`?

Reply:
i believe it should be a string. if that's the case, I'll just wrap it in a `str()`?

Reply:
I think it's worth it for clarity!